src/Rules/DateBetween.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace Spatie\ValidationRules\Rules;
+
+use Carbon\Carbon;
+use Illuminate\Contracts\Validation\Rule;
+
+class DateBetween implements Rule
+{
+    /** @var \Carbon\Carbon */
+    protected $start;
+
+    /** @var \Carbon\Carbon */
+    protected $end;
+
+    /** @var bool */
+    protected $orEquals = false;
+
+    public function __construct(Carbon $start, Carbon $end)
```

Comment:
It would be nice if all these new rules also accept a format of the expected date.